make/platform/Makefile.linux64

```diff
@@ -17,3 +17,5 @@
 
 # Makefile.linux64
 # Special AR and RANLIB definitions for XMT used to be here.
+
+CXXFLAGS += -frtti
```

Comment:
I build 3.6 all the time (with the right part of Makefile.llvm-include commented out) and don't need this change